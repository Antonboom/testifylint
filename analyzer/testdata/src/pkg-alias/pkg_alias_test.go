package pkgalias

import (
	"testing"

	a "github.com/stretchr/testify/assert"
	r "github.com/stretchr/testify/require"
)

func TestPkgAlias(t *testing.T) {
	var predicate bool

	a.Equal(t, true, predicate) // want "bool-compare: use a\\.True"
	r.Equal(t, true, predicate) // want "bool-compare: use r\\.True"
}
