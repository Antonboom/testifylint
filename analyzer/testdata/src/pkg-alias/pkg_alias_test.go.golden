package pkgalias

import (
	"testing"

	a "github.com/stretchr/testify/assert"
	r "github.com/stretchr/testify/require"
)

func TestPkgAlias(t *testing.T) {
	var predicate bool

	a.True(t, predicate) // want "bool-compare: use a\\.True"
	r.True(t, predicate) // want "bool-compare: use r\\.True"
}
